backend/danswer/document_index/vespa/app_config/hosts-dev.xml

```diff
@@ -1,36 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright Vespa.ai. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root. -->
 <hosts>
-  <host name='vespa-configserver-0.vespa-internal.darwin-dev.svc.cluster.local'>
-    <alias>config0</alias>
+  <host name="index">
+    <alias>danswer-node</alias>
   </host>
-  <host name='vespa-configserver-1.vespa-internal.darwin-dev.svc.cluster.local'>
```

Comment:
lets have 3 files,
hosts-local  -> for local testing
hosts-prod  -> for testing on aks cluster on darwin namespace
hosts-dev -> for testing on aks cluster on darwin-dev namespace

and do the if else changes accordingly 





[11:23](https://uipath-product.slack.com/archives/D073CFF246B/p1737006830558279)
and do the if else changes accordingly